i18n/validate.py

```diff
@@ -222,6 +222,12 @@ def add_args(self):
             help="Includes empty translation strings in .prob files."
         )
 
+        self.parser.add_argument(
+            '-f', '--force',
```

Comment:
Nit: I'm not sure, but --check-all might be clearer than --force. I don't feel strongly though, so feel free to leave it as is if you want.